src/styles/card.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: $background-color;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  transition: transform 300ms;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 4px;
+    width: 100%;
+    cursor: pointer;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-dark;
+    transition: color 300ms;
+  }
+
+  &:hover {
+    transition: transform 300ms;
+    transform: scale(1.2);
+
+    .card__title {
+      color: $hover-title;
+    }
+  }
+
+  &__code {
+    width: 100%;
+    color: $text-color-gray;
+    font-size: 10px;
+    line-height: 14px;
+    font-style: normal;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 24px;
+    width: 100%;
+    padding: 0;
+
+    &--text {
+      display: flex;
+      justify-content: flex-end;
+      margin:0;
+      padding: 0;
```

Comment:
You can reset all default styles in `index.scss` or make a separate `reset.scss` file
```suggestion
```

Reply:
delete it in such cases after reset styles

Reply:
Still not fixed